images/linux/Ubuntu2004-README.md

```diff
@@ -128,13 +128,13 @@
 | Tool     | Version       |
 | -------- | ------------- |
 | PHP      | 7.4.24 8.0.11 |
-| Composer | 2.1.8         |
+| Composer | 2.1.9         |
 | PHPUnit  | 8.5.21        |
 ```
     Both Xdebug and PCOV extensions are installed, but only Xdebug is enabled.
 ```
 ### Haskell
-- Cabal 3.6.0.0
+- Cabal 3.4.1.0
```

Comment:
3.6.0.0 and 3.4.0.0 have problems on windows with packages that use autoconf >=2.70.

This was fixed in 3.4.1.0 and 3.6.2.0.

Since there were a couple of issues with 3.6.0.0, I've set the **reccomended* version to 3.4.1.0 until I'm confident 3.6.x is good.

ghcup by default installs the recommended version of a tool, if you omit a version. See:

```sh
# installs recommended version 3.4.0.1
ghcup install cabal
# same
ghcup install cabal recommended
# installs latest 3.6.1.0
ghcup install cabal latest
# installs ver
ghcup install cabal <ver>
```